### DISCUSSION TITLE
Call for users / contributors

### DISCUSSION THREAD

**Discussion by arlyon** [arlyon/async-stripe#168] [Show and tell]
Hey!

We are putting together a site to showcase the project, and would love to feature some people who are using it. If you are using `async-stripe` at your company / in your project, leave a comment here. Looking forward to hearing from you.

Similarly if you are a contributor and would like to be featured let me know. I am making this opt-in as I know people don't always want to be plastered everywhere.

Here is an in-progress deployment: https://stripe-rs.vercel.app/

**Comment by baldwinboy** [comment]
Hi, I've opted to use async-stripe at my company
